server/plugin/kv_test.go

```diff
@@ -82,3 +85,104 @@ func TestStoreUser(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteUser(t *testing.T) {
+	defer monkey.UnpatchAll()
+	s := new(pluginStore)
+	p := Plugin{}
+	s.userKV = kvstore.NewHashedKeyStore(kvstore.NewPluginStore(p.API), UserKeyPrefix)
+	for _, test := range []struct {
+		description   string
+		setupTest     func()
+		expectedError string
+	}{
+		{
+			description: "User is not loaded successfully from the KV store using mattermostID",
```

Comment:
Is there any need for adding successfully in every test cases?
```suggestion
			description: "User is not loaded from the KV store using mattermostUserID",
```